src/domains/translation-resources/translation-resources.service.ts

```
@@ -0,0 +1,425 @@
+
+interface ChapterRange {
+  bookCode: string;
+  startChapter: number;
+  endChapter: number;
+}
+
+function isRecord(value: unknown): value is Record<string, unknown> {
+  return value !== null && typeof value === 'object' && !Array.isArray(value);
+}
+
+type ExtractedAquiferAsset = {
```

ESLint

[error] Use an `interface` instead of a `type`.

(ts/consistent-type-definitions)